### PR TITLE
Create notes for skimage2 coordination meeting on 2024-01-29

### PR DESCRIPTION
**Pull request by lagru**

### CODE REVIEW

**Review by jni — APPROVED** [review]

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Meeting notes "Coordinating skimage2" 
+
+- June 29, 2024 at 20:00 UTC
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
- January 29, 2024 at 20:00 UTC
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- *Realistically, how much resources do we have?*
+  - Paid hours left according to EOSS5 budget (Jan 2024):
+    - Lars: ~1200 h
+    - Marianne: ~250 h
+
+- *How will we communicate about the work?*
+  - Weekly dev update from Lars on discuss.scientific-python.org community list
+
+- *How should work be prioritized?*
+  - There's a new tag: [`🥾 Path to skimage2`](https://github.com/scikit-image/scikit-image/labels/%3Ahiking_boot%3A%20Path%20to%20skimage2)
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - There's a new label: [`🥾 Path to skimage2`](https://github.com/scikit-image/scikit-image/labels/%3Ahiking_boot%3A%20Path%20to%20skimage2)
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  - Use GitHub project (new) or rather a milestone? 
+  - Requesting reviews?
+  - ...?
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
    - Maintainers replied they were fine being requested PR reviews (using GitHub's feature).
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+- *How do we speed-up consensus-seeking and decision making for the new API?*
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
- *How do we speed up consensus-seeking and decision-making for the new API?*
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  - [Clarify objection period for lazy consensus in SKIP 1 #7020](https://github.com/scikit-image/scikit-image/pull/7020)
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - Clarify objection period for lazy consensus in SKIP 1: [#7020](https://github.com/scikit-image/scikit-image/pull/7020)
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  - ...?
+
+- *How can paid maintainers help with bottlenecks?*
+  - Coordinate consensus seeking periods and call votes?
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - Coordinate consensus-seeking periods and call votes?
```
I don't remember whether we retained this option; if so, we can remove the question mark.

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+  - ...?
+
+- Stéfan: Request to core developers, for the six months starting around May:
+  - Please attend weekly meetings where possible
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - Please attend community calls whenever possible
```
Aren't they biweekly (by timezone)?

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+  - Please follow along with 2.0 issues on GH (we have to make decisions rapidly and move on)
+  - Please review as much 2.0 transition code as possible
+
+## Timeline & process
+
+### Starting now
+
+- Make as many [non-breaking changes](https://github.com/scikit-image/scikit-image/wiki/API-changes-for-skimage2#proposed-api-changes-doable-ahead-of-skimage2) as possible
+  - **Note: not many releases to complete deprecations!**
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - **Note: There are not many releases left to complete deprecation cycles!**
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+- Release: `scikit-image==X+1`
+  - *Support & backporting for 1.5 years from here on?*
+- Release: `skimage2`
+- Update third-party documentation resources: StackOverflow, etc.
+
+## Advantages & challenges
+
+**User perspective:**
+
+- Can use `skimage` and `skimage2` a the same time
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
- Can use `skimage` and `skimage2` at the same time 
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+- Don't have to maintain two APIs
+- Decisions & breaking changes during short time frame
+
+---
+
+## Typing scikit-image
+
+- We care about public API first
+- Type description in docstrings will be ground truth
+  - Automatic stub generation from docstrings, e.g [docs2stubs](https://github.com/gramster/docs2stubs)
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - Automatic stub generation from docstrings, e.g, [docs2stubs](https://github.com/gramster/docs2stubs)
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+  - Please review as much 2.0 transition code as possible
+
+## Timeline & process
+
+### Starting now
+
+- Make as many [non-breaking changes](https://github.com/scikit-image/scikit-image/wiki/API-changes-for-skimage2#proposed-api-changes-doable-ahead-of-skimage2) as possible
+  - **Note: not many releases to complete deprecations!**
+- Concretize breaking-changes
+  - e.g. which functions use automatic scaling?
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
  - e.g., which functions use automatic scaling?
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+  - Requesting reviews?
+  - ...?
+
+- *How do we speed-up consensus-seeking and decision making for the new API?*
+  - [Clarify objection period for lazy consensus in SKIP 1 #7020](https://github.com/scikit-image/scikit-image/pull/7020)
+  - ...?
+
+- *How can paid maintainers help with bottlenecks?*
+  - Coordinate consensus seeking periods and call votes?
+  - ...?
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
```

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+
+- *How should work be prioritized?*
+  - There's a new tag: [`🥾 Path to skimage2`](https://github.com/scikit-image/scikit-image/labels/%3Ahiking_boot%3A%20Path%20to%20skimage2)
+  - Use GitHub project (new) or rather a milestone? 
+  - Requesting reviews?
+  - ...?
+
+- *How do we speed-up consensus-seeking and decision making for the new API?*
+  - [Clarify objection period for lazy consensus in SKIP 1 #7020](https://github.com/scikit-image/scikit-image/pull/7020)
+  - ...?
```
Comment (mkcor) [inline meeting-notes/2024/2024-01-29_skimage2-meeting.md]:
```suggestion
```

**Review by mkcor — APPROVED** [review]
Thank you, @lagru! :100:

File: meeting-notes/2024/2024-01-29_skimage2-meeting.md (hunk 7 above)
Comment (lagru) [reply meeting-notes/2024/2024-01-29_skimage2-meeting.md]: Yes, for some. I think America is mostly compatible with both timezone focuses.

**Comment by lagru** [comment]
Thanks for the cleanup @mkcor.

**Pull request merged**